### ISSUE TITLE
Trove Classifier Request

### ISSUE THREAD

**Issue by doctormo** [pypa/trove-classifiers#20]
Request to add a new Trove classifier.

## The name of the classifier you would like to add

* `Framework :: Inkscape`
* `Framework :: Inkscape :: Extensions`

## Why do you want to add this classifier?

The Inkscape project would like to install it's python based extensions using pip/pypi and having a specific classifier would allow our inkscape extension manager graphical user interface to search for and show only Inkscape specific extensions in the results.

Background issue: https://github.com/pypa/packaging-problems/issues/335

The Inkscape project uses the word "extensions", but I see plugins is the typical word used in python land. Either would be fine, extensions for preference.

**Comment by pradyunsg** [comment]
Hi! Thanks for the request. The PyPI moderators have a few guidelines for the creation of new `Framework` classifiers. Because removing classifiers is very difficult, and the longer the classifier list gets the harder it is to browse, the bar to add new ones is relatively high. Currently, we have these guidelines:

* New `Framework` classifiers should be created if they are "notable;" and 
* They will be of immediate use to existing projects (in other words, no creating classifiers simply in anticipation of future uses)

We understand that "notability" is subjective and we'd appreciate help judging that; responses to the second guideline help.

As to the second guideline ('immediate use to existing projects'), the best way to demonstrate notability is to have *10 or more other project maintainers* submit comments on the requesting issue (i.e. here!) about their desire to use the classifier on their published project.

We appreciate any help gathering those responses.

**Comment by doctormo** [comment]
Thanks pradyunsg for the clear requirements. This helps me build a sense of what we need to do in Inkscape to get the classifier recognised. For now we will stick to using the generic ones until we have a notable number of classifiers.

It would be helpful if pypi allowed future classifiers so we could guide creators to use something which would then work right away if this came to pass. But I believe at the moment packages are rejected if they have classifiers that aren't in the single list right?

**Comment by pradyunsg** [comment]
Yes, PyPI rejects uploads that use an invalid / non-existent classifier. You can find the current list of classifiers for PyPI on https://pypi.org/classifiers.

**Comment by doctormo** [comment]
Sorry I mean to say: That behaviour is unhelpful for prospective classifiers, it's rather stringent as it could just ignore them.

**Comment by merwok** [comment]
Could this request be granted or rejected? Although I don’t know if Inkscape has moved on since.

**Comment by di** [comment]
I think we are waiting on:

> For now we will stick to using the generic ones until we have a notable number of classifierees.

@doctormo, is there a notable number of projects that would use this classifier now?

**Comment by doctormo** [comment]
Would use? I think yes. Although none of them are published on pypi as far as I know.

https://inkscape.org/gallery/=extension/

There are even more on GitLab / GitHub where ever there is an inx file with a py file bundled together.

But none of these are currently using pypi as we couldn't make the system work together properly.
